src/Database/Schema/PostgresSchema.php

```diff
@@ -409,17 +409,22 @@ public function columnSql(Table $table, $name)
 
         if (isset($data['null']) && $data['null'] === false) {
             $out .= ' NOT NULL';
+        } elseif (isset($data['null']) && $data['null'] === true) {
+            $out .= ' NULL';
```

Comment:
I made the changes. We could probably also get rid of the "DEFAULT NULL" since it is also implied.